newrelic/terraform-provider-newrelic
# fix: Recreate Synthetics resources on account ID change

# Description

This PR forces a new Synthetics resource if Account ID is changed


## Type of change

Please delete options that are not relevant.


## Checklist:

Please delete options that are not relevant.


## How to test this change?

1. Create a Synthetics monitor in one account
2. Switch to a different account and change the account ID for the monitor
3. Apply again

The Synthetics monitor should be deleted from the original account and recreated in the new account.
